t/06-users.t

```diff
@@ -35,7 +35,7 @@ subtest 'new users are not ops and admins' => sub {
 };
 
 subtest 'system user presence' => sub {
-    my $system_user = $t->app->schema->resultset("Users")->search({username => 'system'})->first;
+    my $system_user = $t->app->schema->resultset("Users")->find({username => 'system'});
```

Comment:
There's no unique constraint (only on a combination with other columns) so this should be limited to one row.

Reply:
The same counts as for the other occurrence when the system user is wanted. Maybe this could be a function in the `ResultSet/Users.pm`.